filament/src/details/Material.cpp

```diff
@@ -192,6 +192,22 @@ Material* Material::Builder::build(Engine& engine) {
         return nullptr;
     }
 
+    MaterialDomain materialDomain;
+    materialParser->getMaterialDomain(&materialDomain);
+    if (materialDomain == MaterialDomain::SURFACE) {
+        StereoscopicType const engineStereoscopicType = engine.getConfig().stereoscopicType;
+        StereoscopicType materialStereoscopicType = StereoscopicType::NONE;
+        materialParser->getStereoscopicType(&materialStereoscopicType);
```

Comment:
Does the bool returned here indicate whether getting the chunk was successful or not? Should we only do the following conditional only if the chunk was successfully parsed? 

I think this'll happen if the material is of an older version than filament.

Reply:
Yeah I also realized it returns a boolean but found other usages don't check that either. Maybe it'd nice to handle them together later.